src/components/shared/NoResult/NoResult.scss

```diff
@@ -0,0 +1,134 @@
+@import "../../../styles/fonts.scss";
+@import "../../../styles/colors.scss";
+@import "../../../styles/variables.scss";
+@import "../../../styles/animations.scss";
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.container {
+    width: $fullWidth;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    padding-top: 120px;
+
+    .image {
+        display: block;
+        margin: 0 auto;
+        width: $fullWidth;
+        max-width: 400px;
+    }
+
+    .bannerButton {
+        width: $fullWidth;
+        max-width: 180px;
+        display: block;
+        margin: 0 auto;
+        margin-top: 10px;
+        background: $white;
+        border: $defaulBorder;
+        border-radius: $defaultBorderRadius;
+        color: $black;
+        padding: 10px 20px;
+        font-weight: $semiBold;
+        font-family: $mainFont;
+        transition: $defaultTransition;
+        animation: $landingBannerAnimation;
+        font-size: $fontsize-16;
+        cursor: pointer;
+
+        &:hover {
+            box-shadow: $textShadowNeon;
+            animation: none;
+        }
+    }
+
+}
+
+.containerText {
+    width: $fullWidth;
+
+    .headText {
+        font-size: $fontsize-32;
+        font-weight: $semiBold;
+        margin-bottom: 10px;
+    }
+
+    .subTitleText {
+        font-size: $fontsize-20;
+        font-weight: $thick;
+    }
+
+}
+
+@media screen and (max-width: $mediaFirstTabletScreen) {
```

Comment:
You have a lot of @media screen, try to refactor to `rem` approach

Reply:
refactored